task/trivy-scanner/0.1/tests/pvc.yaml

```diff
@@ -8,5 +8,5 @@ spec:
     - ReadWriteOnce
   resources:
     requests:
-      storage: 100Mi
+      storage: 200Mi
```

Comment:
any reason for increasing the storage size? Not a blocker, just asking out of curiosity